### PR TITLE
Fixing typo regarding sequence augmentation name

### PR DESCRIPTION
**Pull request by michaelmaillot**
Replace `sequences` by `sequence`

### CODE REVIEW

File: msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md [hunk 1]
```diff
@@ -35,20 +35,24 @@ The Teams AI Library is available in JavaScript and C# languages allowing you to
...
+  - [Responsible AI](#responsible-ai)
+  - [Predictive engine for mapping intents to actions](#predictive-engine-for-mapping-intents-to-actions)
+  - [Action Planner](#action-planner)
+  - [Prompt management](#prompt-management)
+  - [Augmentation](#augmentation)
+    - [Vector data sources](#vector-data-sources)
+  - [Enhanced reasoning](#enhanced-reasoning)
+  - [Feedback loop](#feedback-loop)
+  - [Code samples](#code-samples)
+  - [Next step](#next-step)
```
Comment (v-ypalikila) [inline msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md:56]: Hi @michaelmaillot,

Thank you for contributing to the docs. We accept your suggestion; however, we can't accept this change as we already have a heading Index at the beginning of the article.

image

We will remove all the heading links from this section, and we will merge PR tomorrow.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a73899d22137e309fb77de1b4971a0a57201d4aa/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4107c935-54af-44e5-93b2-f3809b54782d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ef17f4d7514ba35a4ad7cf1d9fc565ebeeb6564a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fba800d1-dac7-4c7c-a303-362e0be9c8b1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3964f95fc48eb92722e23afaeb0b337a5521a69b/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5ac75682-65b6-4dd6-a48d-395e35c1db0c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**